components/loadercontainer/templates.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Templates
+page_title: LoaderContainer Templates
```

Comment:
Rename the article and file to Template. If we start adding more templates it might be better to add separate articles or even a folder. What I used to do for the dropdowns (Putting them all in one article) might not be the best option.